pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -597,21 +597,22 @@ func (service *SecurityPolicyService) createOrUpdateVPCSecurityPolicy(obj *v1alp
 	return nil
 }
 
-func (service *SecurityPolicyService) DeleteSecurityPolicy(obj interface{}, isVPCCleanupOrGC bool, createdFor string) error {
+func (service *SecurityPolicyService) DeleteSecurityPolicy(obj interface{}, isGC, isVPCCleanup bool, createdFor string) error {
```

Comment:
I find that all consumers in this change has set parameter "isGC" as `false`, is it really needed in the public function `DeleteSecurityPolicy`?

Reply:
We set `isGC` as `true` in the networkpolicy and securitypolicy GC function:

https://github.com/vmware-tanzu/nsx-operator/pull/763/files#diff-816bade9860c7cd5eb294711ae56296dc1f400e3c4f546b52368482f01956b00R206

https://github.com/vmware-tanzu/nsx-operator/pull/763/files#diff-f45f721ba06ab477de36df3522bb24718a5f69d5747f3d6b5ae543fe98d68877R380

Reply:
Got it, thanks for helping point it out.